### PR TITLE
Converter for data as supplied by French government

### PR DESCRIPTION
**Pull request by ivorbosloper**

### CODE REVIEW

File: fiboa_cli/datasets/fr.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+ID = "FR"
+TITLE = "Registre Parcellaire Graphique; Crop Fields France"
+DESCRIPTION = """
+France has published Crop Field data for many years. Crop fields are declared by farmers within the Common Agricultural Policy (CAP) subsidy scheme.
+
+The anonymized version is distributed as part of the public service for making reference data available contains graphic data for plots (basic land unit for farmers' declaration) with their main crop. This data has been produced by the Services and Payment Agency (ASP) since 2007.
+"""
+BBOX = [-6.047022416643922, -3.916364769838749, 68.89050422648864, 51.075100624023094]
+
+PROVIDER_NAME = "Anstitut National de l'Information Géographique et Forestière (contact.geoservices@ign.fr )"
```
Comment (m-mohr) [inline fiboa_cli/datasets/fr.py:16]:
```suggestion
PROVIDER_NAME = "Anstitut National de l'Information Géographique et Forestière"
```

File: setup.py [hunk 2]
```diff
@@ -37,6 +37,9 @@ def get_description():
         'nl': [
             "pyogrio"
         ],
+        'fr': [
```
[the remark this answers is not in the document]
Comment (ivorbosloper) [reply setup.py:40]: Thanks for the review, I tried to handle every comment

File: .github/workflows/pytest.yml [hunk 3]
```diff
@@ -24,6 +24,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install -e .[nl]
+          pip install -e .[fr]
```
Comment (m-mohr) [inline .github/workflows/pytest.yml:27]:
```suggestion
          pip install -e .[nl,fr]
```

File: fiboa_cli/datasets/fr.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+The anonymized version is distributed as part of the public service for making reference data available contains graphic data for plots (basic land unit for farmers' declaration) with their main crop. This data has been produced by the Services and Payment Agency (ASP) since 2007.
+"""
+BBOX = [-6.047022416643922, -3.916364769838749, 68.89050422648864, 51.075100624023094]
+
+PROVIDER_NAME = "Anstitut National de l'Information Géographique et Forestière"
+PROVIDER_URL = "https://www.data.gouv.fr/en/datasets/registre-parcellaire-graphique-rpg-contours-des-parcelles-et-ilots-culturaux-et-leur-groupe-de-cultures-majoritaire/"
+# Attribution example as described in the open license
+ATTRIBUTION = "IGN — Original data downloaded from https://geoservices.ign.fr/rpg, updated on June 14 2024"
+LICENSE = {"title": "Etalab-License_Ouverte", "href": "https://etalab.gouv.fr/licence-ouverte-open-licence", "type": "text/html", "rel": "license"}
```
Comment (m-mohr) [inline fiboa_cli/datasets/fr.py:20]:
```suggestion
LICENSE = {"title": "Licence Ouverte / Open Licence", "href": "https://etalab.gouv.fr/licence-ouverte-open-licence", "type": "text/html", "rel": "license"}
```

**Review by m-mohr — APPROVED** [review]
Looks good, thank you.

**Pull request merged**